### DISCUSSION TITLE
If I remove a device how do I add it later?

### DISCUSSION THREAD

**Discussion by simondsmason** [jason0x43/hacs-hubitat#203] [General]
I am only using a few devices in HA from HE but I have a lot of devices in maker for other applications.  So I thought I would remove them from the hacs-hubitat integration to cut down on the noise.  I tested it with one device but I can't see how I would go about adding the device later if I needed it?  Thanks.

(sorry - mean't to make this a discussion but can't see how to move it to discussions!)

**Comment by jason0x43** [comment]
The integration will automatically add any devices exposed by the Maker API instance it's attached to whenever the integration starts up, usually when starting HA. When a device is removed using the integration's config flow, it will be re-added the next time the integration starts up. To fully remove a device, it should be remove from the list of devices exported by Maker API.

Note that you can have multiple instances of Maker API on Hubitat. If you use Maker API for more than just integrating with HA, and different use cases use different sets of devices, you may want to create a instance of Maker API just for integrating with HA.

**Comment by simondsmason** [comment]
Thanks, I like the idea of adding a second instance of maker for this purpose.  Would definitely get rid of the noise.
